ads/openx.md

```diff
@@ -30,11 +30,18 @@ __Required:__
 - `data-auid` - The ad unit ID to display
 - `data-host` - SSL-enabled OpenX delivery domain
 
+__Optional:__
+
+- `json` - Additional json options.
+
+  `customVars` field - please refer to the [documentation](https://docs.openx.com/Content/developers/ad_request_api/custom_variables_in_ad_calls.html).
```

Comment:
I would suggest inline this note to the json param so reader don't think customVars is also an element attribute.